include/stl2/detail/concepts/fundamental.hpp

```diff
@@ -48,25 +48,25 @@ STL2_OPEN_NAMESPACE {
 	}
```

Comment:
Since we're touching this file anyway: rename `FloatingPoint` to `floating_point` and move it out of namespace `ext` now that it's been standardized?